### ISSUE TITLE
Rare crash in barbarian fear automation

### ISSUE THREAD

**Issue by yairm210** [yairm210/Unciv#5857]
Maybe caused by mods which don't contain barbarian civ?

java.lang.NullPointerException: 
  at com.unciv.logic.automation.Automation.afraidOfBarbarians (Automation.kt:159)
  at com.unciv.logic.automation.UnitAutomation.tryFogBust (UnitAutomation.kt:66)
  at com.unciv.logic.automation.UnitAutomation.automateUnitMoves (UnitAutomation.kt:225)
  at com.unciv.logic.automation.NextTurnAutomation.automateUnits (NextTurnAutomation.kt:777)
  at com.unciv.logic.automation.NextTurnAutomation.automateCivMoves (NextTurnAutomation.kt:63)
  at com.unciv.logic.GameInfo.nextTurn (GameInfo.kt:228)
  at com.unciv.ui.worldscreen.WorldScreen$nextTurn$1.invoke (WorldScreen.kt:642)
  at com.unciv.ui.worldscreen.WorldScreen$nextTurn$1.invoke (WorldScreen.kt:634)
  at kotlin.concurrent.ThreadsKt$thread$thread$1.run (Thread.kt:30)

**Comment by xlenstra** [comment]
The crash points to the following line:
```kotlin
if (civInfo.gameInfo.tileMap.continentSizes[startingContinent]!! < civInfo.viewableTiles.count() * multiplier)
```
Seems to me that somehow continent data is initialized wrongly

**Comment by yairm210** [comment]
Resolved by e3fb00d9
